src/TokenApiClient.cs

```diff
@@ -58,17 +58,24 @@ private void SetupClientCredentialsHandler(IClientCredentialsOptions configurati
             }
         }
 
-        public async Task<(string accessToken, string refreshToken)> GetAccessTokenAsync(CancellationToken cancellationToken = default)
+        public async Task<TokenResponse> GetAccessTokenAsync(CancellationToken cancellationToken = default)
         {
             return await RequestAccessTokenAsync(cancellationToken);
         }
 
-        public async Task<(string accessToken, string refreshToken)> RefreshAccessTokenAsync(string refreshToken, CancellationToken cancellationToken = default)
+        public async Task<TokenResponse> RefreshAccessTokenAsync(string refreshToken, CancellationToken cancellationToken = default)
         {
-            return await RequestAccessTokenAsync(cancellationToken);
+            var tokenResponse = await RequestAccessTokenAsync(cancellationToken);
```

Comment:
This doesn't need to be changed. The new access token should still be in the `AccessToken` property. The refresh token is used to get a new access token and isn't used to authenticate with the Repository API

Reply:
Fixed.